bundles/org.openhab.binding.tado/src/main/resources/OH-INF/i18n/tado.properties

```diff
@@ -22,8 +22,8 @@ thing-type.config.tado.home.homeId.label = Home Id
 thing-type.config.tado.home.homeId.description = Selects the Home Id to be used if there is more than one home per account.
 thing-type.config.tado.home.rfcWithUser.label = RFC-8628 with User
 thing-type.config.tado.home.rfcWithUser.description = Determines if the user name is included in the oAuth RFC-8628 authentication
-thing-type.config.tado.home.tadoApiUrl.label = elects the URL to be used for API calls (it's possible to use a proxy)
-thing-type.config.tado.home.tadoApiUrl.description = he base URL used for all API requests. Can be changed to use a proxy.
+thing-type.config.tado.home.tadoApiUrl.label = Selects the URL to be used for API calls (it's possible to use a proxy)
```

Comment:
Not sure if you did this manually or by `mvn i18n:generate-default-translations` but this does not reflect the label from the thing-types.xml file. It should be the _exact_ same.